nautobot/extras/tests/test_api.py

```diff
@@ -1975,6 +1976,72 @@ def test_dry_run_job(self):
         self.assertHttpStatus(response, status.HTTP_200_OK)
 
 
+class NoteTest(APIViewTestCases.APIViewTestCase):
```

Comment:
Are you planning on adding tests for the nested `notes/` endpoint on various objects as well?

Reply:
I thought about it, with the API,  the check-schema fails if the NotesSerializerMixin is used but the view is missing. I could likely add a test. Do we want to do the same as we did with adding Notes? Adding it to the main test cases and calling out the change for Plugins?

Reply:
Thinking here more about functional testing (does the `notes/` nested endpoint return the expected data, formatted the expected way?) which probably doesn't need to be re-tested for every view. I'd be fine if you just added it as a one-off test case, pick a viewset.